tests/ttnn/unit_tests/operations/test_moreh_getitem.py

```diff
@@ -655,3 +717,81 @@ def test_getitem_tilized_five_indices(shape_index_dims, dtype, index_size, row_m
     logger.info(out)
 
     assert passing
+
+
+@skip_for_blackhole("Mismatching on Blackhole, see #12349")
+@pytest.mark.parametrize(
+    "shape_index_dim",
+    (((7, 70), 0),),
+    ids=[
+        "2d_W",
+    ],
+)
+@pytest.mark.parametrize(
+    "dtype",
+    (
+        torch.int32,
+        torch.bfloat16,
+    ),
+    ids=["int32", "bfloat16"],
+)
+@pytest.mark.parametrize(
+    "index_size",
+    (4,),
+)
+@pytest.mark.parametrize(
+    "row_major_index",
+    (False,),
```

Comment:
```suggestion
    [False],
```
When creating test shapes in pytest, it's better to use `lists[]`  instead of `tuples()`.
Sometimes you may want to test with a single shape, and using `()` can lead to discrepancies between the visible code and actual behavior, which can be inconvenient.

For example, if you want to test an input shape of size 32x32:
1. "shape", [[32,32]]
- In this case, there's a single test case, and [32,32] is passed as input.
2. "shape", ((32,32))
- According to Python rules, ((32,32)) is treated the same as (32,32). As a result, two separate inputs, 32 and 32, are treated as individual two test cases.

Reply:
Should I replace all the test case from `tuples()` to `list()` or just the single case one, since there is an `if` statement using `tuple()`,  there must be some modifications to the verification step.

```
if index_dims == (0, 1, 2):
        tt_cpu = x[indices[0], indices[1], indices[2]]
```

Reply:
It would be better to change all cases. The case you mentioned should also work if replaced with `[0,1,2]`.